### PR TITLE
docs: receiver docs

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Please add a detailed description of the change, weather it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: docs/create-basic-receiver-app.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+```html
+<script src="//www.gstatic.com/cast/sdk/libs/caf_receiver/v3/cast_receiver_framework.js"></script>
+```
+
+2.  A script element to load the Kaltura Receiver Player SDK.
+
+
+```html
+<script src="//www.kaltura.com/p/{YOUR_PARTNER_ID}/sp/{YOUR_PARTNER_ID}00/embedPlaykitJs/uiconf_id/{UI_CONF_ID}/partner_id/{YOUR_PARTNER_ID}"></script>
```
Comment (itaykinnrot) [inline docs/create-basic-receiver-app.md]: please change to cdnapisec instead of www

Comment (itaykinnrot) [reply docs/create-basic-receiver-app.md]: remove sp/{YOUR_PARTNER_ID}00 + remove partner_id/{YOUR_PARTNER_ID} we dont need them

Comment (dan-ziv) [reply docs/create-basic-receiver-app.md]: done

**Comment by dan-ziv** [comment]
@AnnaAleksandrowicz 
Can you please review?

Thanks

**Pull request merged**